lib/cib/cib_file.c

```diff
@@ -252,14 +283,9 @@ cib_file_perform_op_delegate(cib_t *cib, const char *op, const char *host,
         return -EINVAL;
     }
 
-    for (int lpc = 0; lpc < PCMK__NELEM(cib_file_ops); lpc++) {
-        if (pcmk__str_eq(op, cib_file_ops[lpc].name, pcmk__str_none)) {
-            operation = &(cib_file_ops[lpc]);
-            break;
-        }
-    }
-
-    if (operation == NULL) {
+    rc = get_operation(op, &operation);
```

Comment:
the op==NULL check could move into get_operation()

Reply:
yeah, although that would change the return code (unless we special-case it below, inside the `if` block). Dunno if that even really matters.

Reply:
Or just return the desired final rc directly from get_operation()

Reply:
Good point. I was trying for a little while to figure out a good way to merge `cib_server_ops` and `cib_file_ops`, and I wanted the get-operation function to be identical in both implementations. Since that didn't work out, I can make them different.